### PR TITLE
Ck feature4 login

### PR DESCRIPTION
**Pull request by fxnggg**
### Feature 4 Tasks

- [x] Generate login component
- [x] Add login component to app routing
- [x] Hide regular header so custom header is shown on the login page
- [x] Create new header for login page
- [x] Create back button for login page header
- [x] Route jb hunt logo/name to home page on click
- [x] Route back button to home page
- [x] Fix styling of login page
- [x] Create judge login button
- [x] Create styling for login button options
- [x] Hide login button when judge is at scoring sheet

### CODE REVIEW

**Review by josi543 — APPROVED** [review]
great work! I am going to wait until it is merged and then I will create my new branch

**Review by alison-nicole — APPROVED** [review]
The login component looks good.

File: src/app/features/login/login.component.html
Comment (alison-nicole) [inline src/app/features/login/login.component.html:13]: Is there where the Judge data is sent over to the back end?

Comment (fxnggg) [reply src/app/features/login/login.component.html:13]: Not really. The way they had it set up was that the judge button routes straight to the scoring sheet. There wasn’t a login or place for them to enter a code or anything to confirm that they’re a judge like on the website they have running right now.

Comment (alison-nicole) [reply src/app/features/login/login.component.html:13]: So it was not implemented in our version?

Comment (fxnggg) [reply src/app/features/login/login.component.html:13]: Correct. I just left it the way they originally did it when they gave us the repo. But we could add in something similar to the code verification they do with the judges on their site that’s running.

Comment (alison-nicole) [reply src/app/features/login/login.component.html:13]: I think there is some implementation about that on the backend, so I'm confused now. I was going to suggest that you could follow the 'Judge' login process for the 'Admins'.

Comment (fxnggg) [reply src/app/features/login/login.component.html:13]: I did think it was weird that there wasn’t any verification when I first tried it out. As for the suggestion, do you mean like a code to enter??

Comment (alison-nicole) [reply src/app/features/login/login.component.html:13]: I considered using their email and comparing it against a list of registered 'Judges' or 'Admins', but a code could work too. In the README, they talk about JWT, and I think that's what they are using to grant authorities. I think this is related to login and authentication.

**Pull request merged**
